test/extended/cli/status.go

```diff
@@ -19,12 +19,12 @@ var _ = g.Describe("[sig-cli] oc status", func() {
 
 	var oc = exutil.NewCLI("oc-status")
 
-	g.It("returns expected help messages [apigroup:project.openshift.io][apigroup:build.openshift.io][apigroup:image.openshift.io][apigroup:apps.openshift.io][apigroup:route.openshift.io]", func() {
+	g.It("returns expected help messages [apigroup:project.openshift.io][apigroup:build.openshift.io][apigroup:image.openshift.io][apigroup:route.openshift.io]", func() {
 		out, err := oc.Run("status").Args("-h").Output()
 		o.Expect(err).NotTo(o.HaveOccurred())
 		o.Expect(out).To(o.ContainSubstring("oc describe buildconfig"))
 
-		out, err = oc.Run("status").Output()
+		out, err = oc.Run("status").Args("--suggest").Output()
```

Comment:
What is the reason of this change?

Reply:
`oc status` prints a different output that does not contain "oc new-app" text when DC API is disabled. `--suggest` puts the text back.